hooks/tk-multi-publish2/basic/nuke_publish_reviewsubmission.py

```diff
@@ -0,0 +1,333 @@
+﻿# Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
Suggest renaming to `submit_for_review.py` and moving into the publish2 app hooks. Is there anything nuke-specific that would prevent that?

Reply:
renamed and moved